src/App.test.js

```diff
@@ -2,27 +2,71 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
 import Main from './components/mainComponent/mainComponent';
+import GameHeader from './components/gameHeaderComponent/gameHeaderComponent';
+import GameSquare from './components/squareComponent/squareComponent';
+import GameBoard from './components/boardComponent/boardComponent';
 import Enzyme from 'enzyme';
 import {shallow} from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
+import { wrap } from 'module';
 
 Enzyme.configure({ adapter: new Adapter() });
 
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<App />, div);
-  ReactDOM.unmountComponentAtNode(div);
-});
-
-it('renders game component without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<Main />, div);
-  ReactDOM.unmountComponentAtNode(div);
-});
-
-it('renders a new page after start button is click', () => {
-  const wrapper = shallow(<Main />);
-  wrapper.find('a').simulate('click');
-  const linkToGame = wrapper.find('.game-page');
-  expect(linkToGame.length).toEqual(1);
+describe('Tic Tac Toe Application', () => {
+
+  it('renders without crashing', () => {
+    const div = document.createElement('div');
+    ReactDOM.render(<App />, div);
+    ReactDOM.unmountComponentAtNode(div);
+  });
+
+  it('renders main component without crashing', () => {
+    const div = document.createElement('div');
+    ReactDOM.render(<Main />, div);
+    ReactDOM.unmountComponentAtNode(div);
+  });
+
+  it('renders a new page after start button is click', () => {
+    const wrapper = shallow(<Main />);
+    wrapper.find('a').simulate('click');
+    const linkToGame = wrapper.find('.game-page');
+    expect(linkToGame.length).toEqual(1);
+  });
+
+  it('renders the game header component without crashing', () => {
+    const gameHeaderDiv = document.createElement('div');
+    ReactDOM.render(<GameHeader />, gameHeaderDiv);
+    ReactDOM.unmountComponentAtNode(gameHeaderDiv);
+  });
+
+  it('renders the board component without crashing', () => {
+    const boardDiv = document.createElement('div');
+    ReactDOM.render(<GameBoard />, boardDiv);
+    ReactDOM.unmountComponentAtNode(boardDiv);
+  });
+  
+  it('renders the square component without crashing', () => {
+    const gameSpaceDiv = document.createElement('div');
+    ReactDOM.render(<GameSquare />, gameSpaceDiv);
+    ReactDOM.unmountComponentAtNode(gameSpaceDiv);
+  });
+
+  it('renders one <Square /> component within the <Board /> componenet without crashing', () => {
```

Comment:
you can just say "renders one Square component within the Board component without crashing"

Reply:
Thanks Andy, just modified my tests to mimic this